lib/internal/util.js

```diff
@@ -286,6 +287,10 @@ function getSystemErrorName(err) {
   return entry ? entry[0] : `Unknown system error ${err}`;
 }
 
+function getSystemErrorMap() {
+  return uvGetFullErrMap();
+}
```

Comment:
I would just do the full implementation here rather than splitting it between `lib/internal/errors` and `lib/internal/util`

Reply:
@jasnell Thanks Yeah I thought about it as well I just thought to keep the convention